flow/raster_cache.cc

```diff
@@ -287,15 +317,19 @@ void RasterCache::TraceStatsToTimeline() const {
   size_t picture_cache_bytes = 0;
 
   for (const auto& item : layer_cache_) {
-    const auto dimensions = item.second.image.image_dimensions();
     layer_cache_count++;
-    layer_cache_bytes += dimensions.width() * dimensions.height() * 4;
+    if (item.second.image) {
+      const auto dimensions = item.second.image->image_dimensions();
+      layer_cache_bytes += dimensions.width() * dimensions.height() * 4;
```

Comment:
Pull out variable as something like `RGBA_BYTES_PER_PIXEL`

Reply:
Actually switched to querying the right number rather than assuming...